akka-remote-tests/src/multi-jvm/scala/akka/remote/RemotingMultiNodeSpec.scala

```diff
@@ -48,4 +63,27 @@ abstract class RemotingMultiNodeSpec(config: MultiNodeConfig)
     }
     deleteFlightRecorderFile()
   }
+
+  /** Identify a user actor by actor name for the given `role`.
+   * {{{
+   *   runOn(first) {
+   *     val a = identify(second, "your-actor")
+   *   }
+   * }}}
+   */
+  def identify(role: RoleName, actorName: String, within: FiniteDuration = 10.seconds): ActorRef =
+    identifyWithPath(role, "user", actorName, within)
+
+  /** Identify an actor by `role / path / name`. Use for system actors as well. */
+  private[akka] def identifyWithPath(
+      role: RoleName,
+      path: String,
+      actorName: String,
+      within: FiniteDuration = 10.seconds): ActorRef = {
+    import system.dispatcher
+    system.actorSelection(node(role) / path / actorName).resolveOne(within).map(probe.send(_, Identify(actorName)))
```

Comment:
that is a complicated way of doing
```
system.actorSelection(node(role) / path / actorName).tell(Identify(actorName), probe.ref)
``

BTW, I don't think `node(role) / path / actorName` will work if `path` is multiple elements like "user/foo/bar".
I think this can work: `node(role) / path.split("/") / actorName`

Reply:
Yes it is one of the things I mean to revisit and simplify.